static/styles/components.css

```diff
@@ -160,11 +160,22 @@
     width: 90px;
     margin: 0px -0.5px;
     text-align: center;
+    text-overflow: ellipsis;
+    white-space: nowrap;
+    overflow: hidden;
+    vertical-align: bottom; /* See http://stackoverflow.com/a/43266155/ */
     padding: 3px 10px;
     background-color: #fff;
     cursor: pointer;
 }
 
```

Comment:
Thank you very much for your help, @brockwhittaker.

I applied these changes:

```diff
diff --git a/static/styles/components.css b/static/styles/components.css
index 4d90927..2e4f405 100644
--- a/static/styles/components.css
+++ b/static/styles/components.css
@@ -152,7 +152,10 @@
 .new-style .tab-switcher {
     font-weight: initial;
     margin: 2px 4px;
-    display: inline-block;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    width: 500px;
 }
```

But the result is this:

![image](https://cloud.githubusercontent.com/assets/7356565/24980371/3349c19a-1fd8-11e7-971e-fb34298d3cc7.png)

Am I missing something?